cluster/src/main/java/org/apache/iotdb/cluster/ClusterIoTDB.java

```diff
@@ -637,11 +640,28 @@ public void disablePrintClientConnectionErrorStack() {
     printClientConnectionErrorStack = false;
   }
 
+  public SyncDataClient getSyncDataClient(Node node, int readOperationTimeoutMS) {
+    SyncDataClient dataClient =
+        (SyncDataClient) clientManager.borrowSyncClient(node, ClientCategory.DATA);
```

Comment:
As `clientManager.borrowSyncClient(node, ClientCategory.DATA);` may return `null`, I think we should do something here, otherwise we'll have an NPE on the upper level calling this function.